Makefile

```diff
@@ -34,3 +36,31 @@ apply-crd:
 	@printf "$(OK_COLOR)==> Applying Kangal CRD to the current cluster $(NO_COLOR)\n"
 	@kubectl delete crd loadtests.kangal.hellofresh.com || true
 	@kubectl apply -f charts/kangal/crd.yaml
+
+protoc:
+	@printf "$(OK_COLOR)==> Compiling ProtoBuf$(NO_COLOR)\n"
+	@if [ -z ${GOOGLEAPIS_DIR} ] || [ ! -d ${GOOGLEAPIS_DIR} ]; then printf "$(ERROR_COLOR)==> GOOGLEAPIS_DIR is not set or does not exist$(NO_COLOR)\n"; exit 1; fi
+	@protoc \
+		--proto_path ${GOOGLEAPIS_DIR} \
+		--proto_path $(CURDIR)/proto \
+		--go_opt=paths=source_relative \
+		--go_out=$(CURDIR)/pkg/proxy/rpc/pb \
+		--go-grpc_out=$(CURDIR)/pkg/proxy/rpc/pb \
+		--grpc-gateway_out=$(CURDIR)/pkg/proxy/rpc/pb \
+		--grpc-gateway_opt=logtostderr=true \
+		--grpc-gateway_opt=paths=source_relative \
+		--openapiv2_out=$(CURDIR)/pkg/proxy/rpc/pb \
+		--openapiv2_opt=logtostderr=true \
+		$(CURDIR)/proto/*/*/*/*.proto
```

Comment:
Do we really need this many wildcards?

Reply:
Unfortunately yes. I think right now I can use direct path to a single file, but `/*/` works only for one level and does not expand to any levels.

Reply:
Can't you use `/**/*.proto`? or do we actually care about the amount of levels?

Reply:
nope

```
❯ make protoc
==> Compiling ProtoBuf
Could not make proto path relative: /Users/vladimir.garvardt/Projects/kangal/proto/**/*.proto: No such file or directory
make: *** [protoc] Error 1
```

Reply:
Maybe set the [shell](https://github.com/hellofresh/vault-secrets-controller/blob/master/Makefile#L177)?

Reply:
Nope, does not work as well and there are a lot of different options in in internet, using `find`, `for`, etc. looks like this path is expanded on the `protoc` app level that is C++ app